src/traits.rs

```diff
@@ -50,7 +50,7 @@ macro_rules! float_trait_impl {
             fn from_str_radix(s: &str, radix: u32)
                               -> Result<Self, ::std::num::ParseFloatError>
             {
-                <$t>::from_str_radix(s, radix)
+                Num::from_str_radix(s, radix)
```

Comment:
Ahh you're right, my bad (getting late here).

Any idea why this is only a problem for the float impl of Num and not the integer impl?